deployment/ansible/roles/gtsite.awscli/tasks/main.yml

```diff
@@ -0,0 +1,9 @@
+---
+- name: Install awscli
+  pip: name=awscli version={{ awscli_version }}
```

Comment:
We have a role that can be used for installing the AWS CLI: https://github.com/azavea/ansible-aws-cli